guardian/support-frontend
# Bumping cmp to version 13.8.0 for MSPS changes.

## What are you doing in this PR?

Bumping the version of CMP that will pick up framework changes for the consent-management-platform to allow for new regulations in the US known as the multi-state privacy agreement.

[**Trello Card**](https://trello.com/c/6qAHXGHq)

## Why are you doing this?

See trello card for all the details you could hope for. 

## How to test

Load the browser in a guest or incognito window and the cookie banner should appear.  Click on the "Yes, I'm happy button" to accept all cookies.  Check the version in support-frontend/package.json matches that which shows in the devtools console window when you paste`window.guCmpHotFix.cmp.version`. 
Check that the Privacy Settings link appears in the footer and that clicking on it brings up the Privacy Manager for you to change your consent.

## How can we measure success?

The cookie banner appears in both contexts and can be interacted with.

## Have we considered potential risks?

No significant change to the UI or data obtained.

## Accessibility test checklist

No visible changes.

## Screenshots

No visible differences.
